apps/assistants/sync.py

```diff
@@ -420,7 +420,7 @@ def remove_files_from_tool(ocs_resource: ToolResources, files: list[int]):
         if file in file_references:
             if ocs_resource.extra.get("vector_store_id") and file.external_id:
                 index_manager = OpenAIRemoteIndexManager(client, index_id=ocs_resource.extra.get("vector_store_id"))
-                index_manager.pluck_file_from_index(file_id=file.external_id)
+                index_manager.delete_file_from_index(file_id=file.external_id)
```

Comment:
@SmittieC I renamed this back to `delete_` since we don't use `pluck` anywhere else and I'm not sure what the difference is.